tests/foreman/api/test_architecture.py

```diff
@@ -27,35 +25,6 @@ class ArchitectureTestCase(APITestCase):
     """Tests for architectures."""
 
     @tier2
-    def test_positive_post_hash(self):
-        """Do not wrap API calls in an extra hash.
-
-        @id: 44654ec5-5211-4326-bcad-9824f36a036f
-
-        @Assert: It is possible to associate an activation key with an
```

Comment:
wrong assert statement. typo here as we are associating arch but activation key pointed. Anyways the whole test no longer required.